### PR TITLE
[W11B01] Added Edge Cases to GenerateModelTest

### PR DESCRIPTION
**Pull request by mbmueller**
Kein Anspruch auf Vollständigkeit, sowie ob diese Edge Cases überhaupt getestet werden auf Artemis.

### CODE REVIEW

File: w11b01/test/pgdp/pengusurvivors/GenerateModelTest.java [hunk 1]
```diff
@@ -76,6 +76,71 @@ public void fullyConnected3on3() {
...
+        int[] procrastinators = {};
+        int[][] friendships = {};
+        FlowGraph graph = PenguSurvivors.generateModel(workaholics, procrastinators, friendships);
+        int maxPairs = graph.computeMaxFlowValue();
+        assertEquals(0, maxPairs);
+    }
+
+    @Test
+    public void nullWorkaholics() {
+        int[] workaholics = null;
```
Comment (MagnusSaurbier) [inline w11b01/test/pgdp/pengusurvivors/GenerateModelTest.java:101]: null cases aren't specified in the task and are not tested by the Artemis tests. I would recommend to leave them out for sake of simplicity.

File: w11b01/test/pgdp/pengusurvivors/GenerateModelTest.java [hunk 2]
```diff
@@ -76,6 +76,71 @@ public void fullyConnected3on3() {
...
+        int[][] friendships = null;
+        FlowGraph graph = PenguSurvivors.generateModel(workaholics, procrastinators, friendships);
+        if (graph != null) {
+            int maxPairs = graph.computeMaxFlowValue();
+            assertEquals(0, maxPairs);
+        }
+    }
+
+    @Test
+    public void friendshipOutOfBounds() {
```
Comment (MagnusSaurbier) [inline w11b01/test/pgdp/pengusurvivors/GenerateModelTest.java:136]: The Artemis tests only give valid inputs. The edge cases aren't tested there anyways.

**Review by MagnusSaurbier — APPROVED** [review]
Null and outOfBounds - Errors aren't tested on Artemis, so these cases aren't necessary. 
The tests do seem fine though.

File: w11b01/test/pgdp/pengusurvivors/GenerateModelTest.java (hunk 1 above)
Comment (Anatoly03) [reply w11b01/test/pgdp/pengusurvivors/GenerateModelTest.java:101]: 👍

**Review by Anatoly03 — CHANGES_REQUESTED** [review]
see above

**Comment by mbmueller** [comment]
Closed as Excercise was already due
